### PR TITLE
ASR merging into clarityeco part II

### PR DESCRIPTION
**Pull request by cdmeister**
This request will merge Kaldi into clarityinc/clarityeco to provide automatic speech recognition capabilities. The folder has been structured according ClarityLab process. Have fun !

### CODE REVIEW

File: asr/template/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#Kaldi#
+[Kaldi](http://kaldi.sourceforge.net/) is a speech recognition toolkit written in C++ that is freely available under the Apache license. 
+
+Kaldi has been wraped in RPC framework using [Apache Thrift](http://thrift.apache.org/). 
+
+Two Versions of Open Ephyra are included in ClarityEco, both of which use Apache Thrift for remote procedure calls. The version in this directory is a stand-alone service for use in any application
```
Comment (jhauswald) [inline asr/template/README.md]: why does this talk about OE?

**Comment by jhauswald** [comment]
- I had to install sox from source to get the `sox.h` header file, apt-get only installs the lib. add the description [link](http://sox.sourceforge.net/)
- milliseconds are printed on the client side, seconds on the server a little odd

Works great although I've built kaldi many times. After you make that edit to the readme, lets merge this in.

**Comment by cdmeister** [comment]
Whoops!! For the sox, there is a dev library its:
sudo apt-get install libsox-dev libsox-fmt-all
I did this and it works. I done both from source and from package manager it works either way. its up to you. I will fix the Readme.

**Comment by jhauswald** [comment]
Can you update the readme so i can merge this in?

**Comment by cdmeister** [comment]
Thanks

**Pull request merged**